packwerk.gemspec

```diff
@@ -47,6 +47,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency("sorbet-runtime", ">= 0.5.9914")
   spec.add_dependency("zeitwerk", ">= 2.6.1")
 
+  # For `RailsLoadPaths`
+  spec.add_dependency("railties")
```

Comment:
I think we use a peer dependency instead because I still think there's a use-case for non-rails now that frameworks like hanami are using Zeitwerk. See example in https://github.com/Shopify/packwerk/pull/285#discussion_r1050106654.

Reply:
Interesting okay.

Reply:
Okay I tried, let me know what you think. I also removed the `rails` dependency.